File: tests/python/thread/viewThread.py
import requests
import json
import sys

jsonBody = {
	"id": 1,
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/viewThread', data = json.dumps(jsonBody), headers = headers)
print(res.json())

File: tests/nodejs/thread/viewThread.js
var request = require('request');

var jsonBody = {
	id: "1",
}

request({
	url: 'http://localhost:3000/viewThread',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});